packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,132 @@
+import { useId } from 'react-id-generator';
+import PropTypes, { element, string } from 'prop-types';
+
+export interface AccordionTriggerProps
+  extends React.HTMLAttributes<HTMLButtonElement> {
+  children: React.ReactElement;
+  heading?:
+    | React.ReactElement
+    | {
+        level: string | undefined;
+      };
+}
+
+const AccordionTrigger = ({ children }: AccordionTriggerProps) => {
+  return <>{children}</>;
+};
+
+interface AccordionContentProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode | React.ReactNode[];
+  className?: string;
+}
+
+const AccordionContent = ({
+  children,
+  className,
+  ...otherProps
+}: AccordionContentProps) => {
+  return (
+    <div className={classnames('Accordion__panel', className)} {...otherProps}>
+      {children}
+    </div>
+  );
+};
+
+interface AccordionProps extends ExpandCollapsePanelProps {
+  children: React.ReactNode;
+}
+
+const Accordion = ({ children }: AccordionProps) => {
+  const childrenArray = React.Children.toArray(children);
+
+  const trigger = childrenArray.find(
+    child => (child as React.ReactElement<any>).type === AccordionTrigger
+  );
+
+  const panelElement = childrenArray.find(
+    child =>
+      typeof child === 'string' ||
+      (child as React.ReactElement<any>).type === AccordionContent
+  );
+
+  const isValid = !!(
+    React.isValidElement(trigger) && React.isValidElement(panelElement)
+  );
+
+  if (!isValid) {
+    console.warn(
+      'Must provide <AccordionTrigger /> and <AccordionContent /> element(s). You provided:',
+      {
+        trigger: trigger,
+        panelElement: panelElement,
+        isValid: isValid
+      }
+    );
+    return null;
+  }
+
+  const elementId = useId();
+
+  return (
+    <div className="Accordion">
+      <ExpandCollapsePanel
+        id={panelElement.props.id || `${elementId}-panel`}
+        className={classnames(panelElement.props.className)}
+        {...panelElement.props.otherProps}
+      >
+        <PanelTrigger
+          iconCollapsed="triangle-right"
+          iconExpanded="triangle-down"
+          className={classnames('Accordion__trigger', trigger.props.className)}
+          aria-controls={panelElement.props.id || `${elementId}-panel`}
+          heading={trigger.props.heading}
+          {...trigger.props.otherProps}
+        >
+          {trigger}
+        </PanelTrigger>
+        {panelElement}
+      </ExpandCollapsePanel>
+    </div>
+  );
+};
+
+Accordion.displayName = 'Accordion';
+AccordionContent.displayName = 'AccordionContent';
+AccordionTrigger.displayName = 'AccordionTrigger';
+
+Accordion.propTypes = {
+  children: PropTypes.node
+};
+
+Accordion.defaultProps = {
+  children: null
+};
+
+AccordionTrigger.propTypes = {
+  children: PropTypes.oneOfType([element, string]),
+  headingLevel: PropTypes.oneOf(['h1', 'h2', 'h3', 'h4', 'h5', 'h6', undefined])
+};
+
+AccordionTrigger.defaultProps = {
+  children: null,
+  headingLevel: undefined
+};
```

Comment:
For functional components, the preferred way to set default values is to use the default parameters of the functional arguments.

```suggestion
```

Reply:
Ugh, I ran into a weird merge conflict at one point and there was some really bizarre stuff happening. I initially did it like you see here with "headingLevel" where it was really declarative/limited (you could do H1-6 or undefined), then ended up doing the more dynamic "heading" one that's mostly in there. Will get this fixed!